src/layouts/Application__c-EDA Application Layout.layout

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Layout xmlns="http://soap.sforce.com/2006/04/metadata">
+    <layoutSections>
+        <customLabel>false</customLabel>
+        <detailHeading>false</detailHeading>
+        <editHeading>true</editHeading>
+        <label>Information</label>
+        <layoutColumns>
+            <layoutItems>
+                <behavior>Readonly</behavior>
+                <field>Name</field>
+            </layoutItems>
+            <layoutItems>
+                <behavior>Required</behavior>
```

Comment:
It's not clear from reading the spec if this field should be required at the database level or just on the page layout. We should confirm this.

Reply:
I thought the reason of requiring fields on layout was because users can decide if they want the fields to be required or not but I am ok with double checking in spec.

Reply:
Correct, I've been avoiding requiring at the database level and only requiring via page layout.